src/compile.js

```diff
@@ -902,16 +795,22 @@ Compiler.prototype.vexpr = function (e, data, augvar, augsubs) {
             mangled = e.attr["$r"]().v;
             mangled = mangled.substring(1, mangled.length - 1);
             mangled = mangleName(this.u.private_, new Sk.builtin.str(mangled)).v;
-            mangled = fixReservedWords(mangled);
-            mangled = fixReservedNames(mangled);
             mname = this.makeConstant("new Sk.builtin.str('" + mangled + "')");
             switch (e.ctx) {
                 case Sk.astnodes.AugLoad:
-                    out("$ret = Sk.abstr.gattr(", augvar, ",", mname, ", true);");
+                    out("$ret = ", augvar, ".tp$getattr(", mname, ", true);");
```

Comment:
This was performance reasons. Since every python now inherits from object we know there will be a tp$getattr method. And since it’s such a hot function calling it directly seemed worthwhile.  It’s similar to how we use tp$call directly in compile code if it exists.